bitrise.yml

```diff
@@ -12,10 +12,8 @@ stages:
     - configure_build_fennec: {}
   stage_2:
     workflows:
-    - build_and_test_1_SmokeTest2: {}
-    - build_and_test_2_SmokeTest: {}
-    - build_and_test_3_SmokeTest3: {}
-    - build_and_test_4_SmokeTest4: {}
+    - L10nScreenshotsTests: {}
```

Comment:
You have to modify the trigger, not this... this workflow depends on the other, so it will fail...
https://github.com/mozilla-mobile/firefox-ios/blob/fd683e939283820488bece393fb483f888a5384f/bitrise.yml#L1553
There use `workflow: L10nBuild` instead of the pipeline
And similar to the pull_request_target_branch: main

Reply:
The portion of L10nScreenshotsTests workflow has not been changed but this line failed. Let me see if the failure is intermittent. 😕 
```
+ curl --location --retry 5 --output l10n-screenshots-dd.zip ''
curl: (3) URL rejected: Malformed input to a URL function
```
https://app.bitrise.io/build/50aad50e-56f8-45d5-8e80-f194b09875f8

Reply:
that workflows depend on L10nBuild, it is not going to work when running it first and isolated

Reply:
As I said above, you may need to change also the https://github.com/mozilla-mobile/firefox-ios/blob/fd683e939283820488bece393fb483f888a5384f/bitrise.yml#L1558